<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1240</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextEdit" name="txeRetInfo">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>250</y>
      <width>381</width>
      <height>271</height>
     </rect>
    </property>
   </widget>
   <widget class="QCustomPlot" name="customPlot" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>381</width>
      <height>241</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnTest">
    <property name="geometry">
     <rect>
      <x>1090</x>
      <y>250</y>
      <width>80</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Test</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spbValueExtBigDelay">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>120</y>
      <width>141</width>
      <height>24</height>
     </rect>
    </property>
    <property name="suffix">
     <string> sysPeri</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSetExtSmallDelay">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>90</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>修改外部小延时</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSetExtBigDelay">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>120</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>修改外部大延时</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spbValueExtSmallDelay">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>90</y>
      <width>141</width>
      <height>24</height>
     </rect>
    </property>
    <property name="showGroupSeparator" stdset="0">
     <bool>true</bool>
    </property>
    <property name="suffix">
     <string> timeBase</string>
    </property>
    <property name="prefix">
     <string/>
    </property>
    <property name="maximum">
     <number>100000</number>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QComboBox" name="cmbChExtSmallDelay">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>90</y>
      <width>72</width>
      <height>23</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>3</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>4</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="cmbChExtBigDelay">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>120</y>
      <width>72</width>
      <height>23</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>3</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>4</string>
     </property>
    </item>
   </widget>
   <widget class="QTextBrowser" name="txbFilePath">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>40</y>
      <width>291</width>
      <height>41</height>
     </rect>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;D:/WZH_QT_Program/measureAfterPulse/afterPulseMeasurement_v2/file/test.txt&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSetFilePath">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>40</y>
      <width>80</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>设置存储路径</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnTdcConnect">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>200</y>
      <width>80</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>连接TDC</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnReadTimeStamps">
    <property name="geometry">
     <rect>
      <x>1080</x>
      <y>210</y>
      <width>80</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>读取时间戳</string>
    </property>
   </widget>
   <widget class="lneUnit" name="lneSysPeri">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>9</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>1G</string>
    </property>
   </widget>
   <widget class="lneUnit" name="lneStartPeri">
    <property name="geometry">
     <rect>
      <x>401</x>
      <y>9</y>
      <width>70</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>1M</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSetStartFre">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>10</y>
      <width>80</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>设置起始频率</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSetSysFre">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>10</y>
      <width>80</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>设置系统频率</string>
    </property>
   </widget>
   <widget class="BtnBool" name="btnStop">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>200</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>STOP</string>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>230</y>
      <width>311</width>
      <height>291</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>扫描探测器延时</string>
     </attribute>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>200</y>
        <width>291</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;探测器延时的修改是通过修改其trigger信号来实现的，trigger信号由AWG5200给出，调节范围为-2--2ns。通过扫描改变trigger的延时可以找到正确的开门位置&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QSpinBox" name="spbSpdDelayStart">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>60</width>
        <height>23</height>
       </rect>
      </property>
      <property name="minimum">
       <number>-2000</number>
      </property>
      <property name="maximum">
       <number>2000</number>
      </property>
     </widget>
     <widget class="QSpinBox" name="spbSpdDelayStop">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>30</y>
        <width>60</width>
        <height>23</height>
       </rect>
      </property>
      <property name="minimum">
       <number>-2000</number>
      </property>
      <property name="maximum">
       <number>2000</number>
      </property>
      <property name="value">
       <number>2000</number>
      </property>
     </widget>
     <widget class="QSpinBox" name="spbSpdDelayInterval">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>30</y>
        <width>60</width>
        <height>23</height>
       </rect>
      </property>
      <property name="maximum">
       <number>500</number>
      </property>
      <property name="value">
       <number>20</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>54</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>Start(ps)</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>10</y>
        <width>54</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>Stop(ps)</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>10</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Interval(ps)</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnScanSPD1Delay">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>60</y>
        <width>80</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>扫描SPD1延时</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnScanSPD2Delay">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>90</y>
        <width>80</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>扫描SPD2延时</string>
      </property>
     </widget>
     <widget class="QLCDNumber" name="lcdSPD1Delay">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>80</width>
        <height>23</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 0);</string>
      </property>
      <property name="value" stdset="0">
       <double>100.000000000000000</double>
      </property>
     </widget>
     <widget class="QLCDNumber" name="lcdSPD2Delay">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>80</width>
        <height>23</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 0);</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnGetDelayBetSPD">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>60</y>
        <width>80</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>SPD相对延时</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="spbSpdDelayTime">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>30</y>
        <width>60</width>
        <height>23</height>
       </rect>
      </property>
      <property name="maximum">
       <number>5000</number>
      </property>
      <property name="value">
       <number>500</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>10</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Time(ms)</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>连续运行</string>
     </attribute>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>54</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>Time(ms)</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="spbRunTime">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>60</width>
        <height>23</height>
       </rect>
      </property>
      <property name="minimum">
       <number>0</number>
      </property>
      <property name="maximum">
       <number>2000</number>
      </property>
      <property name="value">
       <number>250</number>
      </property>
     </widget>
     <widget class="QPushButton" name="btnRunTdcConti">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>261</width>
        <height>161</height>
       </rect>
      </property>
      <property name="text">
       <string>Run Tdc</string>
      </property>
     </widget>
     <widget class="BtnBool" name="btnPlotClear">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>240</y>
        <width>80</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Clear</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>获取符合计数</string>
     </attribute>
     <widget class="QPushButton" name="btnCoincNum">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>80</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>符合计数</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QLCDNumber" name="lcdTimer">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>340</y>
      <width>351</width>
      <height>101</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="formLayoutWidget">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>20</y>
      <width>321</width>
      <height>151</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>SPAD 单道计数</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>超导单道计数</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>符合计数</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLCDNumber" name="lcdSpad">
       <property name="smallDecimalPoint">
        <bool>false</bool>
       </property>
       <property name="digitCount">
        <number>9</number>
       </property>
       <property name="segmentStyle">
        <enum>QLCDNumber::Flat</enum>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLCDNumber" name="lcdSnpd">
       <property name="digitCount">
        <number>9</number>
       </property>
       <property name="segmentStyle">
        <enum>QLCDNumber::Flat</enum>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLCDNumber" name="lcdCoin">
       <property name="digitCount">
        <number>9</number>
       </property>
       <property name="segmentStyle">
        <enum>QLCDNumber::Flat</enum>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>平均光子数</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLCDNumber" name="lcdMu">
       <property name="digitCount">
        <number>9</number>
       </property>
       <property name="segmentStyle">
        <enum>QLCDNumber::Flat</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1240</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>仪器设置</string>
    </property>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionQutagConfig"/>
   <addaction name="separator"/>
   <addaction name="actionTekAWG5200"/>
  </widget>
  <action name="actionQutagConfig">
   <property name="text">
    <string>QutagConfig</string>
   </property>
  </action>
  <action name="actionTekAWG5200">
   <property name="text">
    <string>TekAWG5200</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header location="global">qcustomplot.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>lneUnit</class>
   <extends>QLineEdit</extends>
   <header location="global">lneunit.h</header>
  </customwidget>
  <customwidget>
   <class>BtnBool</class>
   <extends>QPushButton</extends>
   <header location="global">btnbool.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
